config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] Kindly consider defining the root path to `users#index` , so when anyone opens your application renders the user page not the default page for ruby on rails

Reply:
Well resolved ✔️